Supporting Resources/v2.0/Datamodel/Datafiles/Activity I-DM-3-B.yml

```diff
@@ -28,15 +28,15 @@ shortDescription:
 
 #A multi-paragraph description of the activity
 longDescription: |
-  Regularly (at least once per year) revisit the defect management metrics you're collecting and compare the effort needed to collect and track these to the expected outcomes. Make knowledgeable decision about removing metrics which consistently don't bring the expected value. Wherever possible, include and automate verification activities for the quality of the collected data and ensure sustainable improvement if any differences are detected.
+  Regularly (at least once per year) revisit the defect management metrics you're collecting and compare the effort needed to collect and track these to the expected outcomes. Make knowledgeable decision about removing metrics which don't deliver value. Wherever possible, include and automate verification activities for the quality of the collected data and ensure sustainable improvement if any differences are detected.
```

Comment:
I'd still leave "the expected" (or sth like "anticipated") value there. Also metrics which still deliver some value, but less than e.g. the effort spent for tracking should be removed from my POV.

Reply:
Problem with the sentence was that it could be interpreted that you could drop metrics that don't score well. And that should not be the point, right ;-) I'll rephrase.